### DISCUSSION TITLE
Custom Font

### DISCUSSION THREAD

**Discussion by wang4621** [wevm/frog-archived#197] [General]
How can i import a custom font into the image?

**Comment by tmm** [comment]
See https://frog.fm/reference/frog#imageoptions

**Reply by wang4621** [reply]
I'm still confused. importing font needs to be async. How would i do this?

**Reply by tmm** [reply]
`imageOptions` can be an async function.

**Reply by wang4621** [reply]
thanks. but do you know why i get this error. everything is in the correct place
<img width="552" alt="Screenshot 2024-03-25 at 8 04 40 PM" src="https://github.com/wevm/frog/assets/82539201/db80ae49-39a5-4147-af22-99c3da7de743">
<img width="442" alt="Screenshot 2024-03-25 at 8 05 41 PM" src="https://github.com/wevm/frog/assets/82539201/e5bdcaf3-790a-426a-bbcb-ebf735082466">
